src/main/perl/Path.pm

```diff
@@ -147,7 +153,7 @@ sub _reset_exception_fail
     if ($EC->error()) {
         # LC::Exception supports formatted stringification
         my $errmsg = ''.$EC->error();
-        $self->debug(1, "Ignoring/resetting previous existing error: $errmsg");
+        $self->debug(1, "Ignoring/resetting previous existing error$msg: $errmsg");
```

Comment:
it's an additional argument you can pass, mainly to get an idea what method is resetting any previously existing fail or exception. very usefull while debugging.